tests/PySys/smoketest_json_400_samples_20ms/run.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pysys
+from pysys.basetest import BaseTest
+
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+import time
+
+"""
+Roundtrip test C8y 400 samples 20ms delay
+
+Given a configured system with configured certificate
+When we derive from EnvironmentC8y
+When we run the smoketest for JSON publishing with defaults a size of 400, 20ms delay
+Then we validate the data from C8y
+"""
+
+
+class PySysTest(EnvironmentC8y):
```

Comment:
Same comment as for the previous test. A lot of this code duplication can be eliminated with some inheritence. Another alternative to run the same test with varying inputs is to use the pysys mode feature: https://pysys-test.github.io/pysys-test/UserGuide.html#running-tests-in-multiple-modes